app/lib/backend_api_logic/service_extension.rb

```diff
@@ -18,27 +18,20 @@ module ServiceExtension
     end
 
     def first_backend_api
-      @backend_api ||= find_or_create_first_backend_api!
+      @backend_api ||= backend_api_configs.first&.backend_api
     end
 
-    def find_or_create_first_backend_api!
-      return unless account
-      config = backend_api_configs.first || create_first_backend_api_config!
-      config.backend_api
-    end
-
-    private
-
-    def create_first_backend_api_config!
+    def build_default_backend_api_config(endpoint = nil)
```

Comment:
This still looks like `build_or_create_default_backend_api`.

Reply:
Renaming this method will also break https://github.com/3scale/porta/blob/8cecd33c61eed6e724ffcc2108effb89fd913ac0/db/migrate/20190716110520_create_the_backend_apis_of_services.rb#L14